### PR TITLE
Disable PSPs by default

### PR DESCRIPTION
**Pull request by marcsanmi**
Follow up of https://github.com/newrelic/newrelic-infra-operator/pull/226 as it was coming from a fork and I couldn't update it. It just updates the docs by running `helm-docs`.

cc @vuqtran88.

### CODE REVIEW

File: charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled)) }}
+{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled) (.Values.rbac.pspEnabled)) }}
```
Comment (paologallinaharbur) [inline charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml:1]: nit: we could add a unittest to cover this

File: charts/newrelic-infra-operator/Chart.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ sources:
   - https://github.com/newrelic/newrelic-infra-operator
   - https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator
 
-version: 1.0.11
+version: 2.0.0
 appVersion: 0.7.0
```
Comment (paologallinaharbur) [inline charts/newrelic-infra-operator/Chart.yaml:11]: nit: it would be nice to merge all Prs and release a new version if we have time

Comment (paologallinaharbur) [reply charts/newrelic-infra-operator/Chart.yaml:11]: prs regarding dependencies

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        kubernetes-version: [ "v1.17.17", "v1.22.0" ]
+        kubernetes-version: [ "v1.17.17", "v1.22.0", "v1.25.3" ]
```
Comment (paologallinaharbur) [inline .github/workflows/ci.yaml:18]: v1.17.17 over ubuntu-latest will fail 😢 you need to pin 20.04

**Review by paologallinaharbur — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (marcsanmi) [reply .github/workflows/ci.yaml:18]: Just realized it... 😅

File: charts/newrelic-infra-operator/values.yaml [hunk 4]
```diff
@@ -44,6 +44,10 @@ admissionWebhooksPatchJob:
   # - name: tmp
   #   emptyDir: {}
 
+rbac:
```
[the remark this answers is not in the document]
Comment (marcsanmi) [reply charts/newrelic-infra-operator/values.yaml:47]: Just for consistency with the first [PR](https://github.com/newrelic/k8s-metadata-injection/pull/233/files) and with the current chart of [nri-kubernetes](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/values.yaml#L510).

File: charts/newrelic-infra-operator/Chart.yaml (hunk 2 above)
Comment (marcsanmi) [reply charts/newrelic-infra-operator/Chart.yaml:11]: I'll try to merge all the dependabot PRs and release first.

File: charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml (hunk 1 above)
Comment (marcsanmi) [reply charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml:1]: Done in 61abeb473578b14734476296b6a80de6b51b7fd1.

File: charts/newrelic-infra-operator/values.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (paologallinaharbur) [reply charts/newrelic-infra-operator/values.yaml:47]: yeah, the same question could be asked for the other Prs, but I am fine with it. PSP are linked for some reasons to cluster roles (dependency that caused their deprecation :) )

**Pull request merged**